src/ParameterPostProcessor.php

```diff
@@ -24,17 +24,12 @@ class ParameterPostProcessor
      */
     public function __construct(array $parameters)
     {
-        $this->parameters = new ParameterBag($parameters);
+        $this->parameters = $parameters;
     }
 
     public function __invoke(array $config) : array
     {
-        $parameters = $this->parameters;
-
-        // First, convert values to needed parameters
-        $convertedParameters = $this->convertValues($parameters->all());
-        $parameters->clear();
-        $parameters->add($convertedParameters);
+        $parameters = $this->getResolvedParameters();
```

Comment:
Can we move this into try...catch below and then remove `try...catch` inside `getResolvedParameters`?